build/utils/pkg.ts

```diff
@@ -11,7 +11,7 @@ export const getWorkspaceNames = async (dir = projRoot) => {
   return pkgs
     .filter((pkg) => pkg.dir.startsWith(dir))
     .map((pkg) => pkg.manifest.name)
-    .filter((name): name is string => !!name)
+    .filter(Boolean)
```

Comment:
I think it's unnecessary to change.

Reply:
Well, it’s really not a necessary change, but it can make the code look more concise.